package.json

```diff
@@ -218,10 +218,10 @@
     "cross-env": "^7.0.0",
     "css-loader": "^3.6.0",
     "detect-port": "^1.3.0",
-    "electron": "9.1.0",
-    "electron-builder": "^22.7.0",
-    "electron-devtools-installer": "^3.1.1",
-    "electron-rebuild": "^1.11.0",
+    "electron": "7.1.13",
+    "electron-builder": "^22.3.6",
+    "electron-devtools-installer": "^2.2.4",
```

Comment:
Why were these deps downgraded?

Reply:
Reverted back to the exact deps before #2525 breaking changes

Reply:
I see. I dont think deps need to be downgraded since the bug was caused with electron. Ideally we could keep as many deps up to date as possible.